frontend/src/components/Team.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  Card, CardImg, CardTitle, CardText,
+  CardSubtitle, CardBody, Row, Col
+} from 'reactstrap';
+import '../styles/Team.css';
+import data from '../jsonData/TeamData.json'
+
+const cardStyle = {
+  backgroundColor: '#E4A58F',
+  borderWidth: 'thick',
+  borderColor: '#333',
+}
```

Comment:
One thing I just noticed is this. Can you move this to a stylesheet instead.

Reply:
I just centered the bottom two cards and moved the styling to the stylesheet.